scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2903,10 +2972,12 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
    */
   def templateStats(): List[Stat] = statSeq(templateStat)
   def templateStat: PartialFunction[Token, Stat] = {
+    case token if token.is[Ellipsis] =>
+      ellipsis(1, unquote[Stat])
```

Comment:
What about handing `_: Unquote` here in the same way like you handled it in `topStat`?